packages/demo/src/helpers/launchEditor.ts

```diff
@@ -85,7 +85,7 @@ export function launchEditor(
                 );
             },
             loadSmartFilter: async (file: File) => {
-                return smartFilterLoader.loadFromFile(file, false); // TODO: update optimize
+                return smartFilterLoader.loadFromFile(file);
```

Comment:
A general consideration for the Editor-- saving optimized SmartFilters won't work right now. 
If "Edit Optimized" is true (so the editor is using the optimized SmartFilter) then saving will fail because we don't have a serializer/deserializer for the optimized block. For now, maybe we can disable saving when looking at optimized smartfilters in the editor? Unless it'd be easier to make the serializer/deserializer for that block.

Reply:
I started working on making the editor readonly, and it turned out to be a big change. I'm going to remove the feature for launching the editor to view the optimized graph, we can use Spector in the meantime.